central/reportconfigurations/service/common/authorizer.go

```diff
@@ -12,30 +12,21 @@ import (
 var (
 	// Authorizer is used for authorizing report configuration grpc service calls
 	Authorizer = perrpc.FromMap(map[authz.Authorizer][]string{
-		// TODO: ROX-13888 Replace VulnerabilityReports with WorkflowAdministration.
-		or.Or(
-			user.With(permissions.View(resources.VulnerabilityReports)),
-			user.With(permissions.View(resources.WorkflowAdministration))): {
+		user.With(permissions.View(resources.WorkflowAdministration)): {
 			"/v1.ReportConfigurationService/GetReportConfigurations",
 			"/v1.ReportConfigurationService/GetReportConfiguration",
 			"/v1.ReportConfigurationService/CountReportConfigurations",
 		},
-		// TODO: ROX-13888 Replace VulnerabilityReports with WorkflowAdministration.
 		or.Or(
-			user.With(permissions.Modify(resources.VulnerabilityReports), permissions.View(resources.Integration), permissions.View(resources.Access)),
+			user.With(permissions.Modify(resources.WorkflowAdministration), permissions.View(resources.Integration), permissions.View(resources.Access)),
 			user.With(permissions.Modify(resources.WorkflowAdministration), permissions.View(resources.Integration))): {
 			"/v1.ReportConfigurationService/PostReportConfiguration",
 			"/v1.ReportConfigurationService/UpdateReportConfiguration",
 		},
-		// TODO: ROX-13888 Replace VulnerabilityReports with WorkflowAdministration.
-		or.Or(
-			user.With(permissions.Modify(resources.VulnerabilityReports)),
-			user.With(permissions.Modify(resources.WorkflowAdministration))): {
+		user.With(permissions.Modify(resources.WorkflowAdministration)): {
 			"/v1.ReportConfigurationService/DeleteReportConfiguration",
 		},
-		or.Or(
-			user.With(permissions.Modify(resources.VulnerabilityReports), permissions.View(resources.Integration), permissions.View(resources.Access)),
```

Comment:
Need to do some git archeology to figure out why one `Or` branch has `resources.Access` while other don't, but not a blocker here

Reply:
AFAICT it's due to the previous iteration of report configuration using access scopes, hence requiring the `Access` resource. With the postgres release, this will be switched to Collections instead, which won't require the `Access` permission then anymore.